configure

```diff
@@ -544,7 +544,7 @@ fi
 BOOL_OPTIONS=""
 VAL_OPTIONS=""
 
-opt debug 0 "debug mode"
+opt debug 0 "debug mode; you may also want extra --enable-optimize for this as it disable optimization by default"
```

Comment:
First, grammatical typo: "as it disable optimization by default" is not correct.  perhaps you meant "... it *disables* ..." ?

Second: Its pretty long; it gets out to column 114 in my editor, and so it will pretty long in the users terminal. I'm not sure if we have other help text for configure that is so long on one line.

Third: the word "it" in the output is slightly ambiguous; a naive user might think that it refers to the `--enable-optimize` option.  (Yes, i know that would be a silly interpretation.)

Maybe better text would be just:

```
"debug mode; disables optimization unless `--enable-optimize` given"
```